### PR TITLE
Feature/horizontal forms

### LINKED ISSUE #180: Suport for horizontal forms
Ad suport for horizontal forms

### PR DESCRIPTION
**Pull request by David-Esteves**
Added the possibility of setting the form fields in a horizontal style. it also includes the option to set the size of the label and the field itself (default is 2 for the label and 10 for the form for a grid of 12)

To use it 
```
{% include 'arctic/partials/form_field.html' with field=form.fieldx style="horizontal" %}
```

or 

```
{% include 'arctic/partials/form_field.html' with field=form.fieldx style="horizontal" size="3" %}
```
fixes #180

### CODE REVIEW

File: arctic/templates/arctic/base_create_update.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
                 {% include "arctic/partials/form_validation.html" with form=form style="horizontal" %}
             {% endblock %}
             {% block form_fields %}
-                {% include "arctic/partials/form_fields.html" with form=form style="horizontal" %}
+                {% include "arctic/partials/form_fields.html" with form=form style="" %}
```
Comment (Silveron) [inline arctic/templates/arctic/base_create_update.html]: better to remove style="" ?

File: arctic/templates/arctic/partials/form_field.html [hunk 2]
```diff
@@ -15,32 +14,49 @@
...
+                            {% endif %}
 
                             {% if field.field.required %}
                                 {% render_field field|attr:'required' %}
                             {% else %}
                                 {% render_field field %}
                             {% endif %}
 
-                            {{ field.label|safe }}
+                            {% if style == "horizontal"%}
```
Comment (Silveron) [inline arctic/templates/arctic/partials/form_field.html]: I see a lot of if "style == horizontal" statements. maybe better to setup a different template for horizontal forms?

Comment (David-Esteves) [inline arctic/templates/arctic/partials/form_field.html]: I will reorganize the entire thing a little bit

**Comment by David-Esteves** [comment]
The tests have an incompatibility problem with django 1.10

Redid the form_field.html

File: arctic/templates/arctic/partials/form_field.html (hunk 2 above)
Comment (David-Esteves) [inline arctic/templates/arctic/partials/form_field.html]: Reorganized

**Pull request merged**